srcpkgs/libcacard/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'libcacard'
 pkgname=libcacard
-version=2.7.0
+version=2.8.0
 revision=1
-build_style=gnu-configure
+build_style=meson
+configure_args="-Ddisable_tests=true"
```

Comment:
Instead of uniformly disabling tests, consider adding `configure_args+="-Ddisable_tests=true"` inside a `[ -z "$XBPS_CHECK_PKGS" ]` test (see, for example, the `opencv4` template) to allow those tests to run with `xbps-src check` or `xbps-src -Q pkg`.

If the tests fail for other reasons and can't be fixed, that's a different matter.

Reply:
It does give an issue. Gives test/meson.build "softhsm unknown variable" added error in comment in template